src/tools/site-wide-analysis.md

```diff
@@ -184,26 +184,26 @@ If you do not want to use our [shell script](https://github.com/magento-swat/ins
    shasum -a 512 -c launcher.checksum
    ```
 
-1. Create the `swat-agent.env` file with the following contents.
+1. Create the `config.yaml` file with the following contents.
 
    ```config
-   SWAT_AGENT_PREFERRED_APPLICATION_NAME=<Test Inc>
-   SWAT_AGENT_APP_NAME=<APP_NAME>
-   SWAT_AGENT_APPLICATION_PHP_PATH=php
-   SWAT_AGENT_APPLICATION_MAGENTO_PATH=<APPLICATION ROOT e.g: /var/www/html >
-   SWAT_AGENT_APPLICATION_DB_USER=<APPLICATION_DB_USER>
-   SWAT_AGENT_APPLICATION_DB_PASSWORD=<APPLICATION_DB_PASSWORD>
-   SWAT_AGENT_APPLICATION_DB_HOST=<APPLICATION_DB_HOST>
-   SWAT_AGENT_APPLICATION_DB_NAME=<APPLICATION_DB_NAME>
-   SWAT_AGENT_APPLICATION_DB_PORT=3306
-   SWAT_AGENT_APPLICATION_DB_TABLE_PREFIX=<TABLE_PREFIX>
-   SWAT_AGENT_APPLICATION_DB_REPLICATED=<false or true>
-   SWAT_AGENT_APPLICATION_CHECK_REGISTRY_PATH=<TEMPORARY DIRECTORY e.g: /tmp/swat-agent-production >
-   SWAT_AGENT_BACKEND_HOST=check.swat.magento.com:443
-   SWAT_AGENT_RUN_CHECKS_ON_START=1
-   SWAT_AGENT_LOG_LEVEL=error
-   SWAT_AGENT_ENABLE_AUTO_UPGRADE=true
-   SWAT_AGENT_IS_SANDBOX=false
+   project:
+     appname: "Acme Inc"
+   application:
+     phppath: php
+     magentopath: /var/www/html/example.com
+     checkregistrypath: /path/to/swat-agent/tmp
+     issandbox: false
+     database:
+       user: your-adobe-commerce-db-username
+       password: your-password
+       host: 127.0.0.1
+       dbname: your-adobe-commerce-db-name
+       port: 3306
+       tableprefix:
+   enableautoupgrade: true
+   runchecksonstart: false
+   loglevel: error
    ```
 
    {:.bs-callout-info}
```

Comment:
The configuration reference at the bottom of the topic needs to be updated.

https://github.com/magento/devdocs/pull/9367/files#diff-c98a2434a509b4b3353f16a6b326dc7a768a7d4f68e24c494cc358bfc8fa5c36R348

Reply:
This Configuration reference is backwards compatible. It is still allow you to use ENV variables to override, so we decided to keep it.